### PR TITLE
Added documentation for Apprise

### LINKED ISSUE #228: Example of using Apprise with base image.
Hi, I know that Apprise was added to the base image a while back. I can't find any examples of how to use it though (in the context of a borgmatic backup via cron).

I am wanting to migrate away from the msmtp image, as I use apprise on a few other projects for email alerts due to being easier to install and setup in a container than msmtp.

How are you guys using it (just piping the output of borgmatic into apprise?) and could the base README be updated to have an example?

@modem7 I know you are spearheading apprise. Thanks!

### PR DESCRIPTION
**Pull request by jonathan8devs**
Showing a way to use Appwrise in this project. NTFY didn't work for me and I stumbled across ticket #228 and the hint that Apprise is integrated with it. I would like to share this possibility with other users.

Closes #228

### CODE REVIEW

File: apprise/data/borgmatic.d/config.yaml
Comment (grantbevis) [inline apprise/data/borgmatic.d/config.yaml:1]: Should we edit the default config file we provide?

File: apprise/data/borgmatic.d/config.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+consistency:
+    checks:
+        - repository
+        - archives
+    check_last: 3
+    prefix: 'backup-'
+
+hooks:
+    before_backup:
+        - echo "Starting a backup job."
```
Comment (grantbevis) [inline apprise/data/borgmatic.d/config.yaml:31]: Shouldn't this be an apprise command not an echo?

**Review by grantbevis — CHANGES_REQUESTED** [review]
Thanks for the PR but I'm not sure I agree with the approach.

**Comment by jonathan8devs** [comment]
I revisited this topic recently as I am also writing for a German blog. The guide there was getting outdated, and I wanted to update it. The simplest way to set up a regular backup with log handling is as follows:

**Crontab.txt**
```bash
*/5 * * * * PATH=$PATH:/usr/local/bin /usr/local/bin/borgmatic --stats -v 0 > /tmp/backup_run.log
```

**Config.yaml**
```yaml
before_backup:
  - echo "Starting a backup job."
after_backup:
  - echo "Backup created."
  - apprise -vv -t "your-subdomain.example.com - BorgBackup" -b "$(cat /tmp/backup_run.log)" "mailtos://smtp.example.com:587?user=info@example.com&pass=YourSecurePassword&from=server@example.com"
on_error:
  - echo "Error while creating a backup."
```

**Pull request closed without merging**